src/components/pages/Onboarding/Onboarding.tsx

```diff
@@ -56,9 +57,17 @@ type ReduxStateProps = {
   hasValidInviteToken: boolean;
 };
 
+interface DispatchProps {
+  displayDropdownMessage: (
```

Comment:
Maybe we should just make this function signature a type in the dropdown actions so we don't have to repeat it everywhere, but whatevs

Reply:
An alternative is to make `displayDropdownMessage` just take in a `DropdownMessage` argument rather than three sequential args (minus the autogenerated id, or with it optional at least) and then use that as the arg instead. I've been coming around to object args rather than sequential ones as being easier to work with since autocomplete happens and names are unambiguous, but either way works.